.gitignore

```diff
@@ -1,2 +1,7 @@
 target
 Cargo.lock
+
+# Ignore editor files
+*.swp
+.idea
+
```

Comment:
Please add these to your global gitignore instead (e.g., https://gist.github.com/subfuzion/db7f57fff2fb6998a16c). Otherwise, we end up with an entry for every editor/ide in every project.